langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibTableTest.java

```diff
@@ -451,6 +451,24 @@ public void testRemoveThenIterate() {
         Assert.assertTrue((Boolean) returns);
     }
 
+    @Test
+    public void testRemoveEmptyThenIterate() {
```

Comment:
Shall we use a `dataProvider` instead when invoking the functions?
Also, we can do the assertions in ballerina itself?